recipes/libcurl/all/conanfile.py

```diff
@@ -2,12 +2,18 @@
 import os
 import re
 import functools
-from conans import ConanFile, AutoToolsBuildEnvironment, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
 from conan.tools.microsoft import is_msvc
 from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, chdir, copy, download, get, load, replace_in_file, rmdir, save
+from conan.tools.scm import Version
+from conans import AutoToolsBuildEnvironment
+# TODO: Update to conan.tools.apple after 1.51.3
+from conans.tools import is_apple_os, get_env, os_info, run_environment, unix_path
```

Comment:
you can also migrate
- Autotools build
- run_environment logic (with VirtualBuildEnv I guess)
- unix_path

Reply:
I made the migration and removed the use of `run_environment`.

Reply:
Done here: a6319571686f6facd9a5a28837f2e29acc324e07